docs/deploy_windowsdns.md

```diff
@@ -0,0 +1,134 @@
+<h1 align="center" style="margin-top: 0;">Using K8GB on Azure</h1>
```

Comment:
Can we also link this one in the main `README.md` so it will easy to discover and also appear on the website?